Cat.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Image, Text, View } from 'react-native';
+import kitty from './assets/kitty.jpg'
+
+const Cat = () => {
+  return (
```

Comment:
What is your rationale for using functional components over class components?

Reply:
Hi @pdxchance! The only rationale I have is I followed the React Fundamentals tut https://reactnative.dev/docs/intro-react and it seemed to make sense to me. Should I start using class component? Maybe I could rebuild with class component.

Reply:
@lasedrish Its totally up to you what component style you use. I personally like class components but we could use functional. We will be exploring these differences together. Your code looks clean to me. Feel free to merge in your pull request when you are ready.